main/webapp/modules/core/scripts/project/history-panel.html

```diff
@@ -1,4 +1,10 @@
 <div class="history-panel-help" bind="helpDiv">
+  <div id="close-icon">
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40" enable-background="new 0 0 40 40">
+      <line x1="15" y1="15" x2="25" y2="25" stroke="#000" stroke-width="2.5" stroke-linecap="round" stroke-miterlimit="10"></line>
+      <line x1="25" y1="15" x2="15" y2="25" stroke="#000" stroke-width="2.5" stroke-linecap="round" stroke-miterlimit="10"></line>
+    </svg>
```

Comment:
I wonder if we could not instead reuse an existing cross icon from the rest of the software. I would be surprised if we have not got any already. If we need to introduce a new one, I would avoid pasting its SVG code at two different locations in our source code.

Reply:
I didn't find any cross image in the `images `directory (main/webapp/modules/core/images/). 

>I would avoid pasting its SVG code at two different locations in our source code.

I'll take out the redundancy

Reply:
The jquery-ui icon set has an X icon.  I think it's: `<span class="ui-icon ui-icon-close">`